jdaviz/core/template_mixin.py

```diff
@@ -167,13 +170,19 @@ class BaseSelectPluginComponent(BasePluginComponent):
     """
     # default_mode can be one of empty, first, default_text (requires default_text to be set)
     default_mode = 'empty'
+    filters = List([]).tag(sync=True)
```

Comment:
filters used to be implemented for the DatasetSelect only... this PR brings them up into the parent class so the same infrastructure can be used across all Select components.